integration-tests-jvm/controlbus-statistics/src/main/java/org/apache/camel/quarkus/component/controlbus/it/ControlbusRoute.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.controlbus.it;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.jboss.logging.Logger;
+
+@ApplicationScoped
+public class ControlbusRoute extends RouteBuilder {
+
+    private static final Logger LOG = Logger.getLogger(ControlbusResource.class);
+
+    @Override
+    public void configure() throws Exception {
+        from("direct:control")
+                .routeId("control")
+                .process(e -> LOG.info("control:" + e.getMessage().getBody(String.class)));
```

Comment:
If you add `camel-quarkus-log` as a dependency, then you can use the `log` EIP instead of needing a custom `Logger` instance & processor. E.g:

```
.log("control: ${body}");
```

Reply:
That route was already in the test so I don't know why it was written that way, but I swapped it for the log() method.
https://github.com/apache/camel-quarkus/pull/4200/files#diff-eeeab2ce264c21e4c0f0b9328b72f5a4162ca53507adbd9c61518a5cd4001cb9R38